src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/service/transfertoecm/CreateEcmSingleService.java

```diff
@@ -26,8 +27,9 @@ public class CreateEcmSingleService {
     public void sendCreation(SubmitEvent oldSubmitEvent, String accessToken, CreateUpdatesMsg createUpdatesMsg)
         throws IOException {
         TransferToEcmDataModel transferToEcmDataModel = (TransferToEcmDataModel) createUpdatesMsg.getDataModelParent();
-        String caseTypeId = TribunalOffice.isScotlandOffice(transferToEcmDataModel.getOfficeCT())
-            ? officeName : transferToEcmDataModel.getOfficeCT();
+        String caseTypeId =  TribunalOffice.isScotlandOffice(transferToEcmDataModel.getOfficeCT())
+            ? officeName : getCorrectedOfficeName(transferToEcmDataModel.getOfficeCT());
```

Comment:
LGTM but maybe a unit test can be added to test offices with a space?

Reply:
I have thought about it and reasoned that because the only public interface of the class is void and the existing unit test covers it expected behaviour.  As to the office name, how do you check after the method call while you have no return? Any thoughts?